sqlite/src/where.c

```diff
@@ -4024,6 +4024,9 @@ static LogEst whereSortingCost(
   ** Use the LIMIT for M if it is smaller */
   if( (pWInfo->wctrlFlags & WHERE_USE_LIMIT)!=0 && pWInfo->iLimit<nRow ){
     nRow = pWInfo->iLimit;
+#if defined(SQLITE_BUILDING_FOR_COMDB2)
+    nRow *= gbl_sqlite_sortermult;
```

Comment:
Adi added `gbl_sqlite_sorterpenalty` in https://github.com/bloomberg/comdb2/pull/2421. Would you want to take that into account here?

Reply:
That tunable is taking into account in a different section of code...  Unless I'm misunderstanding their (the tunables) intent.